src/shared_utilities/models.jl

```diff
@@ -159,16 +159,6 @@ function make_update_boundary_fluxes(model::AbstractModel)
     return update_boundary_fluxes!
 end
 
-"""
```

Comment:
this was actually not intended/used correctly - we have a `make_update_drivers` defined in src/shared_utilities/drivers.jl instead.